### ISSUE TITLE
How to change default stun/turn servers

### ISSUE THREAD

**Issue by amirhosseinds** [jitsi/docker-jitsi-meet#1594]
Hi i am trying to change default stun/turn servers and i searched a lot and i got nothing . can any one help me how to do it?

**Comment by damencho** [comment]
You can control it through these variables. 

https://github.com/jitsi/docker-jitsi-meet/blob/1cbb8f8cec9b29036cf6f24ebd087b8ea58958ee/prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua#L25

**Comment by amirhosseinds** [comment]
yes i saw it but how can i set TURN_CREDENTIALS ? with this format:
TURN_CREDENTIALS=username:password
TURN_CREDENTIALS=sharedsecret
which one can i use?

**Comment by amirhosseinds** [comment]
and this cannot be set in config.js file? i have set something like this in config file:

```
config.p2p = {
        enabled:true,
        useStunTurn:true,
        stunServers: [
                { urls: 'turn:myserver.com:3478', username: 'username', credential: 'password'}
        ],
        preferH264: true,
        iceTransportPolicy:'relay'
};
```

**Comment by damencho** [comment]
The config file is not good idea to keep credentials as it is public. That's why it is configure in prosody and uses a shared secret. And prosody produces short live credentials passing it to the client to use it.

**Comment by damencho** [comment]
This is what it looks like when configured in coturn https://github.com/jitsi/jitsi-meet/blob/0170c65c7b84979df5b74634134b4e93b0a35c4e/doc/debian/jitsi-meet-turn/turnserver.conf#L4C1-L5C1

**Comment by amirhosseinds** [comment]
thanks bro problem solved

**Comment by andreas1107** [comment]
Hi, how exactly did you solve this issue? I wish to use an external TURN service for Jitsi docker to avoid having to reverse proxy. I have received a username and password to use - how do I set this? 

I have tried in my .env file: 

# TURN server
TURN_CREDENTIALS=sharedsecret
TURN_TRANSPORT=tcp
TURNS_HOST=turnservice.com
TURNS_PORT=443


My second question would be how I test the TURN service - what flag do I set in .env file to force traffic via the TURN service only? Sorry I have searched all over including [here](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#turn-server-configuration) but didn't find the answer. 

Thanks
